src/main/java/com/vdmytriv/bookstoreapp/dto/UpdateBookRequestDto.java

```diff
@@ -1,12 +1,15 @@
 package com.vdmytriv.bookstoreapp.dto;
 
+import jakarta.validation.constraints.Min;
 import java.math.BigDecimal;
 
-public record UpdateBookRequestDto(String title,
-                                   String author,
-                                   String isbn,
-                                   BigDecimal price,
-                                   String description,
-                                   String coverImage) {
-
+public record UpdateBookRequestDto(
+        String title,
+        String author,
+        String isbn,
+        @Min(0)
+        BigDecimal price,
+        String description,
+        String coverImage
```

Comment:
Should you validate other fields? Do you use the full update or only the provided fields?